Makefile

```diff
@@ -67,7 +67,7 @@ fmt:
 lint:
 	$(GOVET) $(PACKAGES)
 	@cat /dev/null > $(LINT_LOG)
-	@$(foreach pkg, $(PACKAGES), $(GOLINT) $(pkg) | grep -v -e thrift-gen -e thrift-0.9.2 >> $(LINT_LOG) || true;)
+	@$(foreach pkg, $(PACKAGES), $(GOLINT) $(pkg) | grep -v -e IndexService.go -e thrift-gen -e thrift-0.9.2 >> $(LINT_LOG) || true;)
```

Comment:
`pkg/es/mock/IndexService.go` is the one causing lint errors, so I added the comment there.